frontend/packages/ceph-storage-plugin/src/components/namespace-store/create-namespace-store.tsx

```diff
@@ -22,8 +22,16 @@ const CreateNamespaceStore: React.FC<CreateNamespaceStoreProps> = ({ match }) =>
         <BreadCrumbs
           breadcrumbs={[
             {
-              name: 'Openshift Data Foundation',
-              path: resourcePathFromModel(ClusterServiceVersionModel, appName, ns),
+              name: appName ? 'Openshift Container Storage' : 'Openshift Data Foundation',
```

Comment:
```suggestion
              name: appName ? 'OpenShift Container Storage' : 'Openshift Data Foundation',
```

Reply:
Not sure about this since we used "Openshift" here earlier. Nonetheless, your suggestion did pop up in my mind when I was writing this, and it does seem to be the right way to go.